apps/microsoft-teams/frontend/test/mocks/mockChannels.json

```diff
@@ -3,24 +3,28 @@
     "id": "19:e3a386bd1e0f4e00a286b4e86b0cfbe9@thread.tacv2",
     "name": "General",
     "teamId": "ed57f808-c14f-4a53-bf53-e36de0783385",
+    "tenantId": "666e56a6-1f2a-47c7-b88c-1ed9e1bb8668",
```

Comment:
Same comment here about the `tenantId`, that wasn't in the original mock data and I'm curious why we added it. Doesn't really hurt anything to keep it 😄

Reply:
same answer from above, happy to remove it if this secondary index isn't being stored and we want to filter it on the call.